test/org/sqldroid/DriverUnitTest.java

```diff
@@ -674,7 +674,7 @@ public void testAutoCommit() throws Exception {
         String jdbcURL = JDBC_URL_PREFIX + dbFile;               
  
         Properties removeLocale = new Properties();
-        removeLocale.put(SQLDroidDriver.ADDITONAL_DATABASE_FLAGS, android.database.sqlite.SQLiteDatabase.NO_LOCALIZED_COLLATORS);
+        removeLocale.put("AdditionalDatabaseFlags", android.database.sqlite.SQLiteDatabase.NO_LOCALIZED_COLLATORS);
```

Comment:
This file will be removed in a future PR. Didn't remove yet to consult previous tests